nio4r.gemspec

```
@@ -6,7 +6,7 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Tony Arcieri"]
   spec.email         = ["bascule@gmail.com"]
   spec.homepage      = "https://github.com/socketry/nio4r"
-  spec.license       = "MIT AND (BSD-2-Clause OR GPL-2.0-or-later)"
+  spec.licenses      = ["MIT", "BSD-2-Clause", "GPL-2.0-or-later"]
```

```suggestion
  spec.licenses      = ["MIT", "BSD-2-Clause"]
```

In our case, we have `license.md` which clarifies the situation, and additionally, if we *were* adopting the GPL from `libev` our own code would also need to be GPL, therefore, we couldn't also release under the `MIT` license. Thus, I think we can simplify this and assume that `spec.licenses` is a union of all licenses.